mslib/mscolab/models.py

```diff
@@ -181,7 +181,7 @@ class Message(db.Model):
     text = db.Column(db.Text)
     message_type = db.Column(db.Enum(MessageType), default=MessageType.TEXT)
     reply_id = db.Column(db.Integer, db.ForeignKey('messages.id'))
-    created_at = db.Column(db.DateTime, default=datetime.datetime.utcnow)
+    created_at = db.Column(AwareDateTime, default=datetime.datetime.now(tz=datetime.timezone.utc))
```

Comment:
try if 
db.Column(AwareDateTime, default=lambda: datetime.datetime.now(tz=datetime.timezone.utc))

makes it not necessary to have the `self.created_at =` line.

also for the other.